doc/man1/fipsinstall.pod

```diff
@@ -0,0 +1,148 @@
+=pod
+
+=head1 NAME
+
+openssl-fipsinstall,
+fipsinstall - perform FIPS configuration installation
```

Comment:
Remove the single `fipsinstall`...  the only reason they exist for *other* commands is backward compatibility.

Reply:
Its in every other single command ..  I assume you mean delete the 
'fipsinstall - perform FIPS configuration installation'

Reply:
Not the whole line.  Basically, I want this:

```
openssl-fipsinstall - perform FIPS configuration installation
```